shared/src/main/scala/mlscript/Typer.scala

```diff
@@ -50,6 +51,7 @@ class Typer(var dbg: Boolean, var verbose: Bool, var explainErrors: Bool)
     private val abcCache: MutMap[Str, Set[Var]] = MutMap.empty
     def allBaseClassesOf(name: Str): Set[Var] = abcCache.getOrElseUpdate(name,
       tyDefs.get(name).fold(Set.empty[Var])(_.allBaseClasses(this)(Set.empty)))
+  
```

Comment:
Ishaaaaaaaaan

Reply:
:lotus_position: let the white space flow through you :wind_chime: :leaves: